byway/src/main/webapp/base.js

```diff
@@ -13,17 +13,21 @@ function setProgressBar(pageNumber){
     items[i].className = 'active';
   }
   let tripKey = getTripKeyFromUrl();
-  document.getElementById('choose-destinations-progress-bar').href = configureTripKeyForPath(tripKey, '/destinations.html');
-  document.getElementById('choose-interests-progress-bar').href = configureTripKeyForPath(tripKey, '/interests.html');
-  document.getElementById('view-route-progress-bar').href = configureTripKeyForPath(tripKey, '/routepage.html');
+  document.getElementById('choose-destinations-progress-bar').href = 
+configureTripKeyForPath(tripKey, '/destinations.html');
+  document.getElementById('choose-interests-progress-bar').href = 
+configureTripKeyForPath(tripKey, '/interests.html');
+  document.getElementById('view-route-progress-bar').href = 
+configureTripKeyForPath(tripKey, '/routepage.html');
 }
 
 /** 
  * Sets up LogoutLink if user logged in
  * Sends alert and redirects to login page if user is not logged in
  */
 function setupLogoutLink(){
-    fetch("/api/login").then(response => response.json()).then((loginStatus) =>{
+    fetch("/api/login").then(response => response.json())
+    .then((loginStatus) =>{
     if (loginStatus.isLoggedIn) {
       let logoutLink = document.createElement("a");
       logoutLink.id = "logout-button";
```

Comment:
Stuff should be intended farther than the braces that delimit it.

Reply:
fixed